synapse/config/room_directory.py

```diff
@@ -54,6 +58,11 @@ def read_config(self, config):
 
     def default_config(self, config_dir_path, server_name, **kwargs):
         return """
+        # Wether the public room list can be searched. When disabled blocks
```

Comment:
"Whether", though I prefer to set the example to `false` and say something like "Uncomment to disable searching the public room list".